dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/servertype/internal/LibertyGradleJEEPublisher.java

```diff
@@ -88,7 +89,8 @@ public void publishModuleAndChildren(int kind, PublishUnit unit, MultiStatus mSt
                             int publishUnitKind = unit.getDeltaKind();
                             if (!isLC && "ear".equals(projectType) && (ServerBehaviourDelegate.ADDED == publishUnitKind || ServerBehaviourDelegate.CHANGED == publishUnitKind)) {
 
-                                publishOnParent(wsServer, moduleProject, config, "package -DskipLibertyPackage=true",
+                            	String[] tasks = new String[] {LibertyGradleConstants.ASSEMBLE_TASK};
```

Comment:
This line isn't quite indented properly.

Reply:
I think git does not show well when there are mixed tabs and spaces as it looks fine in an editor.  I changed all tabs in the file to spaces in the file.